cocotb/scheduler.py

```diff
@@ -326,6 +327,14 @@ def _test_completed(self, trigger=None):
             self._check_termination()
 
     def react(self, trigger):
+        """
+        .. deprecated:: 1.5
+            This function is now private
```

Comment:
Similarly for others, to be more consistent with the rest of the docs.
```suggestion
        .. deprecated:: 1.5
            This function is now private.
```

Reply:
Was this missed or do you prefer no period?

Reply:
Just forgot to save before pushing.